static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -328,7 +328,9 @@ const Segment = styled('span', {shouldForwardProp: isPropValid})<{color: string}
   padding: 1px ${space(0.5)} 0 0;
 `;
 
-const LegendAnimateContainer = styled(motion.div)<{expanded: boolean}>`
+const LegendAnimateContainer = styled(motion.div, {
+  shouldForwardProp: prop => prop !== 'expanded',
```

Comment:
This is going to end up forwarding motion div props now though I think

Reply:
Because the default of shouldForwardProp ignores anything that isn't dom attributes. But now you've said "forward everything except expanded"